migration/mapper.py

```diff
@@ -0,0 +1,636 @@
+  "DPR_PAL_Search_ClearSearchResults",
+  "pausesall",
+  "pausetwo",
+  "pausethree",
+  "bounce",
+  "importXMLs",
+  "createTdForMatch",
+  "createTables",
+  "normalizeLongSearchResult",
+  "showonly",
+  "atiPause",
+  "atiSearchStart",
+  "atiSearchOffline",
+  "findRegEx",
+  "getRegExtSearchString",
+  "searchHistoryXML",
+  "eraseSearchHistory",
+  "sameSearchHistory",
+  "simSearchHistory",
+  "saveSearchHistory",
+  "dictHistoryXML",
+  "eraseDictHistory",
+  "saveDictHistory",
+  "sortaz",
+  "sortStrip",
+  "comparePaliAlphabet",
+  "findSimilarWords",
+  "groupBySimilarity",
+  "removeDuplicatesFromArray",
+  "moveframey",
+  "parseDictURLParameters",
+  "startDictLookup",
+  "pedsearchstart",
+  "pedFullTextSearch",
+  "dppnsearchstart",
+  "dppnFullTextSearch",
+  "mlsearchstart",
+  "multisearchstart",
+  "epdsearchstart",
+  "attsearchstart",
+  "tiksearchstart",
+  "titlesearchstart",
+  "paliRootsearchstart",
+  "sktsearchstart",
+  "sktRootsearchstart",
+  "makeUniqueStringForCaseInsensitiveFS",
+  "paliXML",
+  "toggleDppnTitle",
+  "DPPNXML",
+  "sktRXML",
+  "sktXML",
+  "getAtthXML",
+  "getTitleXML",
+  "getDppnData",
+  "displayDictData",
+  "openFirstDPRTab",
+  "openDPRTab",
+  "findDPRTab",
+  "findDPRTabs",
+  "findDPRTabByLoc",
+  "updatePrefs",
+  "isDPRTab",
+  "giveIDtoTabs",
+  "checkLastTab",
+  "DPRSidebarWindow",
+  "DPRSidebarDocument",
+  "closeDPRSidebar",
+  "openDPRSidebar",
+  "toggleDPRSidebar",
+  "setCurrentTitle",
+  "closeBrowser",
+  "DPRBottomPaneUpdateStyle",
+  "DPRShowBottomPane",
+  "getSelected",
+  "convert",
+  "changeConversion",
+  "clearboth",
+  "sendtoconvert",
+  "sendtoPad",
+  "clearPad",
+  "savePad",
+  "moveframex",
+  "moveframec",
+  "closeCP",
+  "openCP",
+  "cpFlatten",
+  "showHideId",
+  "moves",
+  "moveanf",
+  "scrolldown",
+  "getlink",
+  "go_anchor",
+  "scrollToId",
+  "makeDeclensions",
+  "outputAnalysis",
+  "analyzeword",
+  "findmatch",
+  "manualCompound",
+  "isIndec",
+  "isUncomp",
+  "outputDef",
+  "showShortDef",
+  "conciseChange",
+  "reanalyze",
+  "conjugate",
+  "conjugateNoun",
+  "conjugateVerb",
+  "conjugateIrrNoun",
+  "conjugateIrrVerb",
+  "conjugateWord",
+  "formatConjugation",
+  "getConjugation",
+  "makeGrammarTerms",
+  "typeAndGender",
+  "conjugateWords",
+  "bv",
+  "showBv",
+  "citation",
+  "bvAlert",
+  "translateTextx",
+  "translateText",
+  "arrangeWords",
+  "checkCompatibleNoun",
+  "checkCompatibleVerb",
+  "makeWord",
+  "translateWord",
+  "transMod",
+  "addPlural",
+  "addPhrasePreps",
+  "sortLongerDec",
+  "stripEnglish",
+  "simpleWordTranslation",
+  "showAltTable",
+  "changeAlt",
+  "insertWordByWord",
+  "clearText",
+  "insertConj",
+  "DPR_keypress",
+  "createCommands",
+  "moveFrame",
+  "devO",
+  "dalert",
+  "ddump",
+  "mainInitialize",
+  "installGlobalHandlers",
+  "triggerUpdateCheck",
+  "setupBTForRG"
+]#Object.keys(window).filter(a => !( whitelist.includes(a) || initObjList.includes(a)))
```

Comment:
please add this as a block comment at the start of this array. that way one does not need to scroll down and get that.